README.md

```diff
@@ -153,17 +161,17 @@ k8s-worker
 ## note: if the following variables are not defined, their tasks will be skipped
 ##   - cluster_uuid
 [k8s-master]
-cv01 ansible_host=10.0.0.15
-cv02 ansible_host=10.0.0.16
-cv03 ansible_host=10.0.0.17
+hostname01 ansible_host=10.0.0.15
```

Comment:
Please either use cv## or modify the existing hv## as well and update the actual template file that gets put down.

Reply:
corrected